firmware/console/binary/tunerstudio.h

```diff
@@ -71,7 +70,7 @@ post_packed {
 	short int count;
 } TunerStudioWriteChunkRequest;
 
-class TunerstudioThread : public ThreadController<CONNECTIVITY_THREAD_STACK> {
+class TunerstudioThread : public ThreadController<2 * CONNECTIVITY_THREAD_STACK> {
```

Comment:
Do we need an explanatory comment here?

Reply:
whoops, that was a local change that snuck in